libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/camera/data/MapboxNavigationViewportDataSource.kt

```diff
@@ -382,6 +392,8 @@ class MapboxNavigationViewportDataSource(
             cameraForFollowing = followingCameraOptions,
             cameraForOverview = overviewCameraOptions
         )
+
+        Log.d("MapZoomingTest", "MapboxNavigationViewportDataSource.evaluate(). options: $options, resulting viewportData: $viewportData")
```

Comment:
Let's also log cameraState in the beginning of the method, as it is used in our calculations.